src/store-api/src/region_request.rs

```diff
@@ -661,23 +663,138 @@ impl From<v1::ChangeColumnType> for ChangeColumnType {
 #[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize)]
 pub enum ChangeOption {
     TTL(Duration),
+    TwscMaxActiveWindowRuns(Option<usize>),
+    TwscMaxActiveWindowFiles(Option<usize>),
+    TwscMaxInactiveWindowRuns(Option<usize>),
+    TwscMaxInactiveWindowFiles(Option<usize>),
+    TwscMaxOutputFileSize(Option<ReadableSize>),
+    TwscTimeWindow(Option<Duration>),
+    TwscRemoteCompaction(Option<bool>),
+    TwscFallbackToLocal(Option<bool>),
+}
+
+impl ChangeOption {
+    pub fn to_change_table_option(&self) -> ChangeTableOption {
+        match self {
+            ChangeOption::TTL(duration) => ChangeTableOption {
+                key: "ttl".to_string(),
+                value: humantime::format_duration(*duration).to_string(),
+            },
+            ChangeOption::TwscMaxActiveWindowRuns(runs) => ChangeTableOption {
+                key: "compaction.twcs.max_active_window_runs".to_string(),
```

Comment:
Can we extract these string literals to consts? They appear multiple times in codebase.

Reply:
yep its a good idea